documentation/src/main/asciidoc/introduction/Introduction.adoc

```diff
@@ -614,15 +614,15 @@ We do need to be careful here if our persistence code uses native SQL, or if it
 ====
 
 Whether we're testing against our real database, or against an in-memory Java database, we'll need to export the schema at the beginning of a test suite.
-We _usually_ do this when we create the Hibernate `SessionFactory` or JPA `EntityManager`, and so traditionally we've used a <<automatic-schema-export,configuration property>> for this.
+We _usually_ do this when we create the Hibernate `SessionFactory` or JPA `EntityManagerFactory`, and so traditionally we've used a <<automatic-schema-export,configuration property>> for this.
 
 The JPA-standard property is `jakarta.persistence.schema-generation.database.action`.
 For example, if we're using `Configuration` to configure Hibernate, we could write:
 
 [source,java]
 ----
 configuration.setProperty(AvailableSettings.JAKARTA_HBM2DDL_DATABASE_ACTION,
-                          Action.SPEC_ACTION_DROP_AND_CREATE);
+                          Action.CREATE);
```

Comment:
This is `org.hibernate.tool.schema.Action.SPEC_ACTION_DROP_AND_CREATE` I'm talking about. So I think this is correct.